PowerEditor/installer/nativeLang/english.xml

```diff
@@ -151,7 +151,7 @@
                     <Item id="42050" name="Paste Binary Content"/>
                     <Item id="42037" name="Column Mode..."/>
                     <Item id="42034" name="Column Editor..."/>
-                    <Item id="42078" name="Multi-Select Next Occurenc"/>
+                    <Item id="42078" name="Multi-Select Next Occurrence"/>
```

Comment:
No need to fix this typo, this line will be removed

Reply:
Does it mean I need to remove from Hindi.xml too?

Reply:
As the line will be removed from English.xml, there is no point to add translation for Hindi.xml too. So removed the typo correction and removed respective line from Hindi.xml too.